packages/fuels/tests/harness.rs

```diff
@@ -2243,6 +2250,351 @@ async fn can_handle_sway_function_called_new() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+#[cfg(feature = "fuel-core-lib")]
+async fn can_call_no_arg_predicate_returns_true() -> Result<(), anyhow::Error> {
+    let amount_to_predicate: u64 = 16;
```

Comment:
The tests seem to be doing some repeated setup work. You could extract that into a separate function to avoid duplication.